sci-mathematics/scilab/scilab-5.5.2-r1.ebuild

```diff
@@ -0,0 +1,298 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+JAVA_PKG_OPT_USE="gui"
+VIRTUALX_REQUIRED="manual"
+
+inherit autotools bash-completion-r1 check-reqs eutils flag-o-matic \
+	fortran-2 java-pkg-opt-2 pax-utils toolchain-funcs virtualx xdg-utils
+
+DESCRIPTION="Scientific software package for numerical computations"
+HOMEPAGE="http://www.scilab.org/"
+SRC_URI="http://www.scilab.org/download/${PV}/${P}-src.tar.gz"
```

Comment:
```suggestion
SRC_URI="https://www.scilab.org/download/${PV}/${P}-src.tar.gz
https://github.com/gentoo/sci/commit/4c2a07c4629c61395a998633ccfcb34d72569529.patch -> ${P}-bug15107.patch"
```

Reply:
Ooh no wait, that is the wrong URL

Reply:
I am doing

Reply:
This is the correct URL: https://raw.githubusercontent.com/efferre79/sci/4c2a07c4629c61395a998633ccfcb34d72569529/sci-mathematics/scilab/files/scilab-5.5.2-bug15107.patch

Reply:
and there is another problem, the patch from github creates a patch file in ${WORKDIR} that has to be applied in src_prepare(), that is an easy fix, the only drawback is that the user will see the message

`* Applying scilab-5.5.2-bug15107.patch ...`